Forgiveness

krisfield opened the pull request:
Adds special page:ClearPendingReviews to allow for users with userright:clearreview the ability to clear pending reviews without the need of using the terminal

jamesmontalvo3 commented:
WatchAnalytics has never had a release before. I just cut an arbitrary `0.1.0` and am in the process of making Meza track that version. Once that's part of Meza's `master` and `32.x` branches, and there's a `32.2.2` version released, then we can look into this change and #77.

FYI, I also made Meza track specific versions of every other extension EMW maintains.

jamesmontalvo3 commented on modules/clearpendingreviews/ext.watchanalytics.clearpendingreviews.js:
```diff
@@ -2,15 +2,10 @@ $( function ()  {
 	$(document).ready( function () {
 		$('.oo-ui-inputWidget-input').change(
```
Do this on the `keyup` or `keydown` rather than `change`. Otherwise if I adjust the time in one field but don't move the focus off of that field, I still have the "clear reviews" button rather than the "preview" button.

jamesmontalvo3 replied:
When I added this comment it wasn't part of my official github review. Not sure if that matters. Adding this sub-comment to see if that ties it into the review.

krisfield commented on specials/SpecialClearPendingReviews.php:
```diff
@@ -0,0 +1,238 @@
+
+			$table .= "</ul>";
+			$table .= "</td>";
+			$table .= "<td>";
```
incorporated vertical align

krisfield commented:
I incorporated all of your feedback except for the timeout limit, trying to think of best way to handle that.

I also added Hooks to the Preview and Clear Pages actions so we could tie in slackbots or whatever we want with those
